qiskit/providers/ibmq/jupyter/dashboard/dashboard.py

```diff
@@ -246,24 +358,44 @@ def _add_job(job: IBMQJob) -> None:
             Args:
                 job: Job to start watching.
             """
+            # Get job status and type
             status = job.status()
-            queue_info = job.queue_info()
-            position = queue_info.position if queue_info else None
-            est_time = queue_info.estimated_start_time if queue_info else None
-            job_widget = create_job_widget(self, job,
-                                           job.backend().name(),
-                                           status.name,
-                                           position,
-                                           est_time)
-            self.jobs.append(job_widget)
+            job_type = get_job_type(job)
+
+            # Create a widget for the new job and add to list
+            if job_type == JobType.IBMQ:
```

Comment:
It could be either, but I need to use JobType as a param later in the function in `clear_done()`. I could move that code into the if/else but then it would be duplicate. They should roughly equate, but the current way, I believe, reduces total code length by 1 or 2 lines.